_pages/docs/next-ls/editors.md

```diff
@@ -181,6 +181,35 @@ language-servers = ["nextls"]
 command = "path/to/next-ls"
 args = ["--stdio=true"]
 ```
+### Development
```

Comment:
```suggestion
### TCP
```

Reply:
I named this heading `Development` in order to maintain cohesion with the heading used in [the README.md file in the next-ls repo](https://github.com/elixir-tools/next-ls?tab=readme-ov-file#development). I understand how `TCP` can make more sense, but aren't you afraid some users will fail to make the connection between `Development` and `TCP` if `Development` isn't mentioned here? Maybe there's an other way than the heading the make the connection explicit.

Reply:
The perspective of this section of the docs is on editor configuration, not contributing. The word development doesn't necessarily imply developing Next LS. It could mean developing your application code.